src/plugins/equalheight/reports/1-acr-wcag21.json

```
@@ -0,0 +1,494 @@
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/euqalheight-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-03-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG21:link-purpose-in-context",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/euqalheight-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-03-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG21:multiple-ways",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/euqalheight-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-03-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG21:headings-and-labels",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/euqalheight-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-03-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG21:focus-visible",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/euqalheight-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-03-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG22:focus-appearance-minimum",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/euqalheight-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-03-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+    {
+      "acr:requirement": "WCAG22:page-break-navigation",
+      "acr:conformance": "acr:satisfied",
+      "acr:assessment": [
+        "https://wet-boew.github.io/wet-boew/demos/acr/demo-wcag21/euqalheight-assessment-wcag21.json"
+      ],
+      "earl:subject": "_:subject",
+      "dct:description": "",
+      "dct:modified": "2023-03-02",
+      "earl:mode": "earl:manual",
+      "@type": [ "acr:ConformanceRequirement" ]
+    },
+}
```

Missing a few conformance requirement.

JSON parsing error:
```suggestion
    }
  ]
}
```